src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -55,6 +62,85 @@ const ApplicationLauncher = _.flow(
   const runtime = getCurrentRuntime(runtimes)
   const runtimeStatus = getConvertedRuntimeStatus(runtime) // preserve null vs undefined
 
+  const FileOutdatedModal = ({ onDismiss, bucketName }) => {
+    const handleChoice = _.flow(
+      withErrorReportingInModal('Error setting up analysis file syncing')(onDismiss),
+      Utils.withBusyState(setBusy)
+    )(async shouldCopy => {
+      await Promise.all(_.flatMap(async analysis => {
+        const currentMetadata = analysis.metadata
+        const file = analysis.name.split('/')[1]
+        if (shouldCopy) {
+          currentMetadata[hashedOwnerEmail] = ''
+          await Ajax().Buckets.analysis(googleProject, bucketName, stripExtension(file), tools.RStudio.label).copyWithMetadata(stripExtension(file).concat(`_copy${Date.now().toString()}.`).concat(tools.RStudio.ext), bucketName, currentMetadata)
+        }
+        currentMetadata[hashedOwnerEmail] = 'doNotSync'
+        await Ajax().Buckets.analysis(googleProject, bucketName, stripExtension(file), tools.RStudio.label).updateMetadata(file, currentMetadata)
+      }, outdatedAnalyses))
+      onDismiss()
+    })
+
+    const getDisplayList = analyses => {
+      let res = ''
+      let i = 1
+      const end = analyses?.length
+      for (const analysis of analyses) {
+        res += analysis?.name.split('/')[1]
+        if (i < end) {
+          res += ', '
+        }
+        i++
+      }
+      return res
+    }
+
+    return h(Modal, {
+      onDismiss,
+      width: 530,
+      title: outdatedAnalyses?.length > 1 ? 'R Markdown Files In Use' : `R Markdown File Is In Use`,
+      showButtons: false
+    }, [
+      Utils.cond(
+        [outdatedAnalyses?.length > 1, () => [p(`These R markdown files are being edited by another user and your versions are now outdated. Your files will no longer sync with the workspace bucket.`),
+          p(getDisplayList(outdatedAnalyses)),
+          p('You can'),
+          p(['1) ', b('save your changes as new copies'), ' of your files which will enable file syncing on the copies']),
```

Comment:
We use `strong` for emphasizing text semantically:

```suggestion
          p(['1) ', strong(['save your changes as new copies']), ' of your files which will enable file syncing on the copies']),
```

Similarly below.